apps/bookmark/js/bookmark_editor.js

```diff
@@ -87,9 +119,14 @@ var BookmarkEditor = {
     var title = this.bookmarkTitle.value.trim();
     this.saveButton.disabled = title === '';
   },
+  
+  _installApp: function bookmarkEditor_installApp() {
+    window.navigator.mozApps.install(this.manifestURL);
+  },
 
   save: function bookmarkEditor_save(evt) {
     this.saveButton.removeEventListener('click', this.saveListener);
+    this.installAppButton.removeEventListener('click', this._installApp);
```

Comment:
it was added using binding, please save the reference to this._installApp.bind(this) to remove this listener, thanks a lot